lib/cinematic.ts

```diff
@@ -652,7 +652,7 @@ class Cinematic {
         window.addEventListener('resize', () => this.handlePlayerResize());
         this.handlePlayerResize();
 
-        if (ResizeObserver) {
+        if (window.ResizeObserver) {
             new ResizeObserver(() => this.handlePlayerResize()).observe(this._container);
```

Comment:
Warum oben mit `window.` und unten ohne?

Reply:
Eigentlich braucht man das `window` nicht (das ist ja der Global Scope) wenn man auf globale Variablen zugreift in Browser-JS, aber wenn man checken will ob etwas im globalen Scope definiert ist muss man mit `window.XYZ` prüfen, sonst bekommt man nen Error falls es nicht vorhanden ist. Alternativ könnte man auch `window.hasOwnProperty('ResizeOvserver')` checken oder n try-catch drumrum machen. Das mit dem `if (window.XYZ) { ... }` ist aber der gängigste und kürzeste Weg (wird zB auch oft bei Oplyfills verwendet)